### PR TITLE
Switch from travis to GitHub actions

### PR DESCRIPTION
**Pull request by JeffreyThiessen**
Test actions completed here https://github.com/JeffreyThiessen/staramr/pull/1/checks?check_run_id=1552589014

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [![Build Status](https://travis-ci.org/phac-nml/staramr.svg?branch=development)](https://travis-ci.org/phac-nml/staramr)
+[![Build Status](https://github.com/phac-nml/staramr/workflows/Integration%20Tests/badge.svg?branch=master&event=schedule)](https://github.com/phac-nml/staramr/actions?query=branch%master)
```
Comment (apetkau) [inline README.md:2]: Can you remove the TravisCI build status badge?

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks great. Thanks so much.

Other than the one comment, do you happen to know why it didn't trigger GitHub Actions?

**Comment by apetkau** [comment]
I can actually see all the actions are up on https://github.com/JeffreyThiessen/staramr/actions

File: README.md (hunk 1 above)
Comment (JeffreyThiessen) [reply README.md:1]: Oops, fixed now.

**Comment by JeffreyThiessen** [comment]
> I can actually see all the actions are up on https://github.com/JeffreyThiessen/staramr/actions

Yeah it appears to be a security thing with Pull requests from forked repo's that change actions so I linked the test actions on my fork a the top.

**Comment by apetkau** [comment]
Looks good. Merging this in.

**Review by apetkau — APPROVED** [review]

**Pull request merged**